bridge_adaptivity/bridge_lti/static/bridge_lti/js/bridge-lti.js

```diff
@@ -0,0 +1,11 @@
+(function ($) {
+    $(document).ready(function () {
```

Comment:
```As of jQuery 3.0 the recommended way to add a ready handler is the first method, $(fn)```
Change `s/$(document).ready(function ()/$(function()/`

For details please take a look at the documentation https://jquery.com/upgrade-guide/3.0/#deprecated-document-ready-handlers-other-than-jquery-function